hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/types/ClusterStateResponse.java

```diff
@@ -84,6 +84,12 @@ public final class ClusterStateResponse {
   @JsonProperty("keys")
   private long keys;
 
+  /**
+   * Total count of keys marked for deletion in the cluster.
+   */
+  @JsonProperty("deletedKeys")
+  private long deletedKeys;
```

Comment:
This is including only leaf level (keys/files) respectively for LEGACY/OBS/FSO buckets. Pls include to show deleted directories separately which should be shown if any deleted directories...Just keep in mind that this is only showing the number and not much informative from debugging purpose, so to supplement this, deleted keys/ deleted directories and files inside that directories should be done as new requirement and part of OM DB insights.

Reply:
Sure thing, I will add it to the response as well !!